requirements-dev.in

```diff
@@ -0,0 +1,4 @@
+-r requirements.txt
```

Comment:
FYI: in other packages we have used:
```
-c requirements.txt
```

This is because the requirements.txt files are now considered "constraint files".

from `pip install --help`
```
-c, --constraint <file>     Constrain versions using the given constraints file. This option can be used multiple times.
```